src/core/lib/channel/metrics.h

```diff
@@ -335,7 +336,8 @@ class GlobalStatsPluginRegistry {
     // the lifetime of the callback; when the returned object is
     // destroyed, the callback is de-registered.  The returned object
     // must not outlive the StatsPluginGroup object that created it.
-    std::unique_ptr<RegisteredMetricCallback> RegisterCallback(
+    GRPC_MUST_USE_RESULT std::unique_ptr<RegisteredMetricCallback>
```

Comment:
Please add a comment here saying that multiple callbacks may be registered for the same metric, as long as no two callbacks report data for the same set of labels.